src/settings/_color.scss

```diff
@@ -1,36 +1,58 @@
 // --- Colors --- //
 
```

Comment:
all dark and light colors where created even if not used now. The reason is that they are defined in the color palette https://app.zeplin.io/project/57c02a80c1caf50b0d2e851e/screen/598af71c138655ba087ce28c

The cost of creating there is almost nothing